.template/addons/crud/lib/templates/slim/scaffold/_form.html.slim.tt

```diff
@@ -0,0 +1,28 @@
+= form_with(model: <%= model_resource_name %>) do |form|
+  - if <%= singular_table_name %>.errors.any?
+    div style="color: red"
+      h2 = "#{pluralize(<%= singular_table_name %>.errors.count, "error")} prohibited this <%= singular_table_name %> from being saved:"
+      ul
+        - <%= singular_table_name %>.errors.each do |error|
+          li = error.full_message
+
+<% attributes.each do |attribute| -%>
+  .mb-3
+<% if attribute.password_digest? -%>
```

Comment:
Should we add an indentation for the `if else` block as it makes us a bit confused with the loop above? 💭

Reply:
Since we are using Slim template, the indentation of the html block is important. Thus in this case we cannot indent `.mb-3`. If we are to indent the `if else` block it will look like this:

```
  <% attributes.each do |attribute| -%>
  .mb-3
  <% if attribute.password_digest? -%>
 ```

It also looks a bit weird and confusing for me 😅 . Thus I decided any line that I will put them all the beginning of the line.